game/src/Autoload/SFX.gd

```diff
@@ -0,0 +1,20 @@
+extends Node
+
+var loadedSFX = {}
+
+func _ready():
+	var dir = DirAccess.open("res://audio/sfx/")
+	for fname in dir.get_files():
+		if fname.get_extension() == "ogg":
+			loadedSFX[fname.split(".")[0]] = load("res://audio/sfx/" + fname) # SND-10
```

Comment:
split will fail produce weird results if a file happens to have a period in it, prefer `get_basename()`
```suggestion
			loadedSFX[fname.get_basename()] = load("res://audio/sfx/" + fname) # SND-10
```

Reply:
done in 60d8b55